nservicebus/azure-service-bus/retries.md

```diff
@@ -5,36 +5,47 @@ tags:
 - Cloud
 - Azure
 - Transport
-reviewed: 2016-09-21
+reviewed: 2016-09-22
 ---
 
-This article describes the relationship between the NServiceBus retry behavior and the Azure Service Bus native retry behavior.
-
-NServiceBus supports Immediate Retries (previously known as First Level Retries) and Delayed Retries (previously known as Second Level) at the endpoint instance level as described in the [recoverability](/nservicebus/recoverability/) article.
+This article describes the relationship between [recoverability behavior](/nservicebus/recoverability/) and the Azure Service Bus native retry behavior.
 
 Azure Service Bus supports a `MaxDeliveryCount` at the entity level, which defines how many times Azure Service Bus attempts to deliver a message before sending it to the dead letter queue. Refer to [the full configuration API](/nservicebus/azure-service-bus/configuration/full.md#controlling-entities-queues) article to learn how to adjust this setting.
 
 
-## Immediate Retries vs MaxDeliveryCount
+## Immediate Retries and MaxDeliveryCount
+
+To implement Immediate Retries, the native Azure Service Bus `MaxDeliveryCount` property of the `BrokeredMessage` is used to track the processing attempts by the endpoint instances for a specific message. If the number of attempts exceeds the `MaxRetries` setting, then the message will be forwarded to the delayed retries if configured, and eventually to at the error queue in case error persists. In case Immediate Retries are disabled, the transport will default `MaxDeliveryCount` to 10 attempts.
```

Comment:
> should it be the delayed retries queue?

Nope. Delayed retries queue is a) implementation detail b)not relevant for ASB

> to at the error queue

👍 done

> If immediate retries are disabled in the endpoint

Going to use "If immediate retries are disabled for the endpoint"